README.md

```diff
@@ -28,14 +28,14 @@ public class RunCukesTest {
 
 # How to configure octane-cucumber-jvm in your project:
 
-## For Cucumber 4.2
+## For Cucumber 4.3
```

Comment:
I'm not sure what we should write here. 4.2? 4.3? just 4? 4.2 / 4.3?
@micheleh WDYT?